django_freeradius/base/admin.py

```diff
@@ -141,16 +142,32 @@ def has_delete_permission(self, request, obj=None):
             return False
         return super().has_delete_permission(request, obj)
 
-    def delete_selected_batch(self, request, queryset):
+    def delete_selected_groups(self, request, queryset):
```

Comment:
See here: https://stackoverflow.com/questions/10814972/override-delete-selected-and-update-fields-of-child-model

Maybe we can simply name it `delete_selected`. What do you think?

Reply:
I had tried to overwrite it but with the introduction of [admin.E130](https://docs.djangoproject.com/en/2.2/ref/checks/#modeladmin) we can't name any other action 'delete selected' and I had no choice but to rename the custom action.